config/config.yaml

```diff
@@ -246,3 +246,4 @@ PgNotifier:
   retriggerCount: 500
   trackBatchInterval: 2s
   maxAttempt: 3
+EnableMultitenancy: true
```

Comment:
we don't need the old config, just the support for the old config in our code

Reply:
removed.